src/tutorials/0006/03.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div>
+        <MultipleChoiceLesson
+        :text="text"
+        :question="question"
+        :choices="choices" />
+    </div>
+</template>
+
+<script>
+import MultipleChoiceLesson from '../../components/MultipleChoiceLesson'
+import text from './03.md'
+
+const question = 'How can we identify the encoding used on specific data when using CIDs?'
+
+const choices = [
+  {
+    answer: 'CIDs have a multicodec prefix that specifies in which encoding is the data represented',
+    correct: true,
+    feedback: 'That\'s right!'
+  },
+  {
+    answer: 'All encodings make it safe to try and check the first bytes of the data to figure out which encoding was used',
```

Comment:
This is the problem we are trying to solve: we can stop making assumptions (incorrect ones, unsafe ones) about the encoding used, and actually just read the multicodec identifier from the CID to understand what was the encoding used.

Reply:
I'm not sure I understand this either.

Reply:
Mentioned in the lesson:
```
Usually, applications solve this issue by trying to guess the encoding based on its content, but this is not a deterministic or safe way to do it.
```

Multicodec tries to solve this problem as well with multicodec right?

Reply:
I just chatted with @alanshaw about this distinction @zebateira  has been making between the trustworthiness of a CID versus the trustworthiness of magic numbers in normal encoded data/files used to guess at the file types. It sounds like he takes your point, and that it's true that if you always use CIDs in IPFS, you'll always know how to decode them without fumbling around. However, he thinks that referencing the way applications determine file types will make people think that this is about a different sort of encoding than it's about within IPFS. (We're not discovering that it's a PDF, we're discovering that it's dag-pb). So his recommendation is to remove the comparison to avoid people confusing the two processes with each other. (Feel free to correct me if I'm misrepresenting you, Alan.)

Reply:
sounds good 👌